src/mca/psquash/base/base.h

```diff
@@ -0,0 +1,73 @@
+/* -*- C -*-
+ *
+ * Copyright (c) 2019      IBM Corporation.  All rights reserved.
+ * Copyright (c) 2019      Mellanox Technologies, Inc.
+ *                         All rights reserved.
+ *
+ * $COPYRIGHT$
+ *
+ * Additional copyrights may follow
+ *
+ * $HEADER$
+ *
+ */
+#ifndef PMIX_PSQUASH_BASE_H_
+#define PMIX_PSQUASH_BASE_H_
+
+#include <src/include/pmix_config.h>
+
+#ifdef HAVE_STRING_H
+#include <string.h>
+#endif
+
+#include "src/class/pmix_pointer_array.h"
+#include "src/mca/mca.h"
+#include "src/mca/base/pmix_mca_base_framework.h"
+
+#include "src/mca/psquash/psquash.h"
+
+
+BEGIN_C_DECLS
+
+/**
+ * Sizeof by PMIx type integer values.
+ *
+ * r - return status code
+ * t - type (pmix_data_type_t) of integer value
+ * s - size of type in bytes
+ * (see a comment to `pmix_bfrops_pack_flex` for additional details)
+ */
+#define PMIX_SQUASH_TYPE_SIZEOF(r, t, s)                    \
```

Comment:
Question: Should this macro be part of `bfrops` instead of `psquash`?

I originally put it here since the flex component was the primary user, but looking at it now I think it might be better associated with `bfrops`. I'm fine either way.

Reply:
I moved this macro because it is used only for `psquash` and is not used for `bfrops`, so there is no need to keep it in `bfrops`.

Reply:
That makes sense. Thanks